apps/web/pages/availability/[schedule].tsx

```diff
@@ -60,13 +58,18 @@ export default function Availability({ schedule }: { schedule: number }) {
   }, [data, isLoading, reset]);
 
   const updateMutation = trpc.useMutation("viewer.availability.schedule.update", {
-    onSuccess: async ({ schedule }) => {
-      await utils.invalidateQueries(["viewer.availability.schedule"]);
-      await utils.refetchQueries(["viewer.availability.schedule"]);
-      await router.push("/availability");
+    onSuccess: async ({ prevDefaultId, currentDefaultId, ...data }) => {
+      if (prevDefaultId && currentDefaultId) {
+        if (prevDefaultId !== currentDefaultId) {
```

Comment:
Nice. I think it would be good to add a comment here explaining that you're updating the old default schedule to update it's default state.

Reply:
has this been resolved?

Reply:
No not yet, only commented this just now.

Reply:
@JeroenReumkens pushed some changes with the comments